records_mover/db/loader.py

```diff
@@ -47,6 +47,12 @@ def temporary_loadable_directory_loc(self) -> Iterator[BaseDirectoryUrl]:
         with TemporaryDirectory(prefix='temporary_loadable_directory_loc') as dirname:
```

Comment:
Loader is an abstract class made concrete by each of the database types which can load out of a records directory.